rhel7/overlays/stig_overlay.xml

```diff
@@ -1,965 +1,2 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<overlays xmlns="http://checklists.nist.gov/xccdf/1.1">
-  <overlay disa="1496" owner="disastig" ownerid="RHEL-07-010010" ruleid="rpm_verify_permissions" severity="high">
-    <VMSinfo SVKey="86473" VKey="71849" VRelease="2"/>
```

Comment:
The change here to `stig_overlay.xml` makes it so that every "title" is duplicated 6 times: each "overlay" is duplicated twice, and within each "overlay", each "VMSinfo" is duplicated twice, plus the title is placed once more next to each "VMSinfo".  This is really hard to see because this change also strips out all the whitespace and makes it one VERY long line.